test/integration/render-tests/globe/globe-transition/tile-corners-outside/style.json

```diff
@@ -4,7 +4,7 @@
     "test": {
       "width": 512,
       "height": 512,
-      "allowed": 0.0015
+      "allowed": 0.05
```

Comment:
Same as above, this very high allowed is acceptable since this test is checking for a missing tile.